api/src/test/java/org/openmrs/module/atomfeed/api/XMLParseServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs.module.atomfeed.api;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.module.atomfeed.api.filter.FeedFilter;
+import org.openmrs.module.atomfeed.api.impl.XMLParseServiceImpl;
+
+import javax.xml.bind.JAXBException;
+
+public class XMLParseServiceTest {
+
+	private static final String XML;
+	private static final String FILTER = "filterValue";
+	private static final String BEAN_NAME = "beanName";
+
+	private XMLParseService parseService = new XMLParseServiceImpl();
+
+	@Test
+	public void testCreatingTagFromXML() throws JAXBException {
+		FeedFilter feedFilter = parseService.createFeedFilterFromXMLString(XML);
+		Assert.assertEquals(BEAN_NAME, feedFilter.getBeanName());
+		Assert.assertEquals(FILTER, feedFilter.getFilter());
+	}
+
+	@Test
+	public void testCreatingXMLFromTag() throws JAXBException {
+		FeedFilter feedFilter = new FeedFilter(BEAN_NAME, FILTER);
+		String xml = parseService.createXMLFromFeedFilter(feedFilter);
+		Assert.assertEquals(XML, xml);
+	}
+
+	static {
```

Comment:
Wouldn't it be clearer if the initialization would be stuck to the declaration?

Reply:
For multiline statics, I prefer this to not waste space at the top of the class.

Reply:
ok